lib/paymentrails/Client.rb

```diff
@@ -24,6 +24,11 @@ def delete(endPoint)
       send_request(endPoint, 'DELETE')
     end
 
+    def delete_multiple(endPoint, body)
```

Comment:
It'd be better to keep a single delete method that accepts either a string or array type, but adding it separate for the sake of backwards compat

Reply:
@zero-bites Since this update will introduce a breaking change any way, let's include both these methods and deprecate the old one:
* one single `delete` method, as you mentioned, for the future of the SDK
* keep the old delete methods but mark them deprecated with a comment.

In the same Google Sheet where we did gap analysis, lets please create new sheets to keep a log of changes, we'll compile Release Notes with them.

Reply:
Sounds good, and it's possible to keep backwards compatibility 👍falls under the category of "forward change"; just need to ensure there are no unintended side-effects downstream

I'll implement this in a backwards compatible fashion

Reply:
@Sheikh-Aman I've added these changes to: https://github.com/zero-bites/ruby-sdk/pull/1/commits/9e0caecf7be21e30bf6811ff421227d7efab330d